### PR TITLE
Feature/tnw new responsive

### PR DESCRIPTION
**Pull request by ccobasi**
# Clone of The Next Web

 Version Control, Github Flow
 Introduction to HTML5 & CSS3.
 Position, Flex, Grid, Floats, RWD
 Font, Image, Page Size manipulation with new CSS3 features.

### CODE REVIEW

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] in short, the project is missing a lot of things, i'm going to list a brief list of what's missing, please make the website looks exactly like the original one, and test the project before asking for a review
- [x] fix linter issues
- [x] fix your live version in your README.md
- [x] make more commits, and make sure they are descriptive.
- [x] The black navbar is _not_ black, please fix it
- [x] the desktop version is completely broken and looked like a mobile version, please fix all sections (footer is fine, but make it's width 100% of the viewport width so it doesn't stay shrank as right now)
- [x] add all categories in the category section
- [x] the categories section doesn't 
- [x] the last funding section should be 4 cards on desktop and reduced to 3 on tablet and mobile
- [x] make the layout and spacings more similar to the website
- [x] your mobile version is completely broken as well
- [x] make sure your sections are centered like in the website. (example the categories section)
- [x] make sure you use DRY code (DRY = Don't Repeat Yourself), for example, you used `display: flex` 10 times, you can instead make a class named `flex` and use it on any element you're turning into it.
- [x] again, please make sure you **clone** the website, follow the layout, spacing and design, _then_ ask for a re-review.

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: style.css [hunk 1]
```diff
@@ -0,0 +1,911 @@
...
+.funding-box-title {
+  display: flex;
+  flex-direction: column;
+}
+
+.funding-red-title {
+  background: #f42;
+  color: #fff;
+  padding: 10.6666666667px 16px 10.6666666667px 84px;
+  margin: 0 !important;
```
Comment (umairarshadbutt) [inline style.css]: Please remove `!important`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,984 @@
...
+      </section>
+
+      <section class="most-popular">
+        <div class="grid-container-4">
+          <div>
+            <div class="img-popular1">
+              <div class="title-text">
+                <h2>Hard Fork<span>&nbsp;>&nbsp;</span></h2>
+              </div>
+              <img src="img/12.jpg" alt="hard-fork" style="width: 100%" />
```
Comment (umairarshadbutt) [inline index.html]: Remove inline styling from all the HTML document

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @ccobasi 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Live link is not working in the README file.
- [x] Screenshot image is missing in the README file.
- [x] This branch has conflicts please resolve them.
- [x] Remove inline styling from all the HTML document.
- [x] **The desktop version is completely broken and looked like a mobile version, please fix all sections (footer is fine), Please follow the previous TSE's suggestion**
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by Georjane — CHANGES_REQUESTED** [review]
## Status: Changes Required 🛑 🔧 
### Hi there Team ✋🏽 
Very nice work you have done with the TNW website clone. You are almost there. You have just the following few changes to make to get this project approved:

- [x] Please make sure your Readme file has only one live link to the project
- [x] For the navigation bar, in the large screen size, you have just one navigation row instead of two. It contains the main navigation row and a sub-navigation row.
- [x] For the top 3 articles section, for large screen sizes too, It contains TWN logo at the top left corner of the row ([link to page](https://thenextweb.com/)) which disappears for medium size screen.

This will be all for now. Please submit for another code review when you are done.

_Feel free to reach out to me on Slack if you have any questions, Witah Georjane_ 🤝 
Good luck on your project and Happy Coding 💻 

**Witah Georjane
Code Reviewer at Microverse Inc
[portfolio](https://witahgeorjane.tech/) | [linkedIn](https://www.linkedin.com/in/witah-georjane/)**

**Review by Georjane — APPROVED** [review]
## Status: PR Approved ✔️ ✔️
### Hi Team 🙌
Great job on implementing the required changes. It's a thumbs up for me. 👍 
You can go ahead and merge this PR. 

Goodluck on your next project and Happy Coding 💻

**Pull request merged**